x-pack/plugins/encrypted_saved_objects/server/config.ts

```diff
@@ -37,7 +53,8 @@ export function createConfig$(context: PluginInitializerContext) {
       }
 
       return {
```

Comment:
__note:__ the config object we return here is supposed to be a convenient wrapper around config related stuff used by plugin internally, so it's not required to reflect raw config shape exactly, so I reshaped it slightly.